### ISSUE TITLE
Use advanced compilation step for reduced payload size

### ISSUE THREAD

**Issue by microsoftsam** [Microsoft/pxt-blockly#69]
https://developers.google.com/blockly/guides/modify/web/advanced describes how to use the blockly offline compiler to make use of tree shaking.

**Comment by pelikhan** [comment]
Yes! We should not be relying on a cloud service for a basic compile step.
